api/envoy/api/v2/route/route_components.proto

```diff
@@ -150,6 +150,14 @@ message VirtualHost {
   // independently (e.g.: values are not inherited).
   RetryPolicy retry_policy = 16;
 
+  // [#not-implemented-hide:]
+  // Specifies the configuration for retry policy extension. See the supported retry policy
+  // implementations for further documentation. Note that setting a route level entry will take
+  // precedence over this config and it'll be treated independently (e.g.: values are not inherited).
+  // :ref:`Retry policy <envoy_api_field_route.VirtualHost.retry_policy>` should not be set if this
+  // field is used.
+  google.protobuf.Any retry_policy_extension = 19;
```

Comment:
This should probably be `retry_policy_typed_config` for consistency with the rest of the API.

Reply:
Done.